### ISSUE TITLE
Drum pad right-click context menus not available via keyboard

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32784]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Shortcuts for Drum pad can be reassigned by right clicking on them.  This does not seem to work with keyboard.

STEPS:
1.  Create a score with a drum kit and navigate to that staff.
2. Press Tab several times to get on to the Percussion panel and to any of the Drum Pads.
3. Right click and see a context menu appears.
4. Press right arrow to move to the next Drum Pad item and this time try using the application key, Shift F10 or number pad multiply to emulate a right click.  Notice that there seems to be no way to bring up this same context menu with keyboard.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
Thanks for this report. There is another way to open the context menu, though: when you're already focused on a drum pad, if you press tab an additional time it will say "[name of pad] options". Then you can press space or enter and it will open the context menu.

While focused on one of these "options" buttons, you can use the arrow keys to navigate the drum pad list the same way you can do so when focused on the main pads.
